src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/TelemetryServiceIntTest.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.personrecord.service
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.personrecord.config.IntegrationTestBase
+
+class TelemetryServiceIntTest : IntegrationTestBase() {
+
+  @Test
+  fun `should log correlation id in telemetry events`() {
+    val telemetryLogs = telemetryRepository.findAll()
```

Comment:
This test assumes there is a telemetry log in the repository to begin with. If you ran this in isolation it would fail

Reply:
Probably best to check if it is added to a event for Probation, Prison event. Then you can check if it is the same throughout